zerver/views/development/design_testing.py

```diff
@@ -28,7 +28,23 @@ def dev_buttons_design_testing(request: HttpRequest) -> HttpResponse:
     context = {
         "background_colors": background_colors,
         "icons": get_svg_filenames(),
+        "page_is_design_testing": True,
+        "design_component": "buttons",
+        "doc_root_title": "Button styles browser",
         # We set isolated_page to avoid clutter from footer/header.
         "isolated_page": True,
     }
     return render(request, "zerver/development/design_testing/buttons.html", context)
+
+
+def dev_banners_design_testing(request: HttpRequest) -> HttpResponse:
```

Comment:
`design_components_sandbox_banners`

Reply:
Addressed with the "showroom" naming convention following the discussion at CZO.